migrations/20210816102828_view_ibc_channels.up.sql

```diff
@@ -1,20 +1,23 @@
 CREATE TABLE view_ibc_channels (
     id BIGSERIAL,
-    channel_id VARCHAR NOT NULL,
+    channel_id VARCHAR NOT NULL UNIQUE,
     port_id VARCHAR NOT NULL,
     connection_id VARCHAR NOT NULL,
     counterparty_channel_id VARCHAR NOT NULL,
     counterparty_port_id VARCHAR NOT NULL,
+    counterparty_chain_id VARCHAR NOT NULL,
+    status BOOLEAN NOT NULL,
     packet_ordering VARCHAR NOT NULL,
     last_in_packet_sequence BIGINT NOT NULL,
     last_out_packet_sequence BIGINT NOT NULL,
     total_transfer_in_count BIGINT NOT NULL,
     total_transfer_out_count BIGINT NOT NULL,
     total_transfer_out_success_count BIGINT NOT NULL,
     total_transfer_out_success_rate NUMERIC NOT NULL,
+    created_at_block_time BIGINT NOT NULL,
+    created_at_block_height BIGINT NOT NULL,
     last_activity_block_time BIGINT NOT NULL,
     last_activity_block_height BIGINT NOT NULL,
     bonded_tokens JSONB NOT NULL,
```

Comment:
Can you add also a `verified` field to the table?

Reply:
Sure, no problem!

I think `verified` should be a boolean and `description` should be a string.
`verified` will be initialized as `false`, `description` will be initialized as an empty string.

I assume such fields will be manually updated by DB admin. Feel free to correct me if I am wrong 🙏